src/components/StartButton.js

```diff
@@ -1,12 +1,23 @@
-import newQuote from '../utils/quoteGenerator';
 import '../css/App.css';
+import newQuote from '../utils/quoteGenerator';
 
 const StartButton = (props) => {
```

Comment:
A nice trick with the props (which is the point I saved for this) is that it's most common to deconstruct the `props` object in the parameters here. It means that you don't need to go around calling `props.setQuoteProp`, and it actually makes it easier for other developers to know what props to pass down into this component, otherwise it's just a guessing game. So for example: 

```js
const StartButton = ({setQuoteProp, currentQuoteProp}) => {...}
```

which can be used on line 12 just as `setQuoteProp(quote)`

Reply:
Cheers, this makes total sense!